src/domains/gps-common/02_security.tf

```diff
@@ -54,6 +54,12 @@ data "azuread_service_principal" "iac_principal" {
   display_name = format("pagopaspa-pagoPA-iac-%s", data.azurerm_subscription.current.subscription_id)
 }
 
+# apiconfig subscription key secret
+data "azurerm_key_vault_secret" "gpd_apiconfig_subkey" {
+  name         = "gpd-payments-apiconfig-subkey"
```

Comment:
how is define `gpd-payments-apiconfig-subkey` on kv ?
manually by portal ?

Reply:
Yes, it is the purpose. Can we follow this schema?

`resource "azurerm_key_vault_secret" "gpd_payments_apiconfig_subscription_key" {
  name         = "gpd-payments-subscription-key"
  value        = "<TO_UPDATE_MANUALLY_BY_PORTAL>"
  content_type = "text/plain"

  key_vault_id = module.key_vault.id
  
 # lifecycle
 [...]
 }`

Reply:
yes this is one way